src/Uno.UWP/Storage/Streams/DataWriter.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using System.IO;
+
+namespace Windows.Storage.Streams
+{
+	public partial class DataWriter : IDataWriter, IDisposable
+	{
+		// only one will be not null - depends on constructor used
+		private MemoryStream _memStream = null;
+		private IOutputStream _outputStream = null;
+		// z: fileOutputStream = detachStream();

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5527039)

Reply:
this is not code